packages/reference/src/resources/useResourceLinkActions.ts

```diff
@@ -0,0 +1,104 @@
+import { useCallback, useMemo } from 'react';
+
+import { FieldExtensionSDK } from '@contentful/app-sdk';
+import { EntryProps, KeyValueMap } from 'contentful-management';
+import noop from 'lodash/noop';
```

Comment:
Can we just do an empty function instead of importing lodash?

Reply:
I've removed `noop` reference :)

Reply:
I've removed imports of lodash :)